apis/placement/v1alpha1/stagedupdate_types.go

```diff
@@ -37,21 +38,21 @@ type StagedUpdateRun struct {
 	Status StagedUpdateRunStatus `json:"status,omitempty"`
 }
 
-// StagedUpdateRunSpec defines the desired the update strategy and the snapshot indices
-// on which the update is based.
+// StagedUpdateRunSpec defines the desired rollout strategy and the snapshot indices of the resources to be updated.
+// It specifies a stage-by-stage update process across selected clusters for the given ResourcePlacement object.
 type StagedUpdateRunSpec struct {
 	// A reference to the placement that this update run is applied to.
 	// There can be multiple active update runs for each placement but
 	// it's up to the devOps to make sure they don't conflict with each other.
 	// +kubebuilder:validation:Required
 	PlacementRef PlacementReference `json:"placementRef"`
 
-	// The resource snapshot index that this update run is based on.
+	// The snapshot index of the selected resources to be updated across clusters.
```

Comment:
Could you give a concrete example on what you mean by the snaptshot index?

Something like:

e.g., for resource snapshots "xxx-123-1" and "xxx-123-2", "123" is the index.

Reply:
I noticed that you used policysnaptshot name instead of index below. Is that because there are N resource snapshot for a single version of selected resources?

Reply:
yes, for each resource snapshot index, there are N resource snapshots. The index is "123" in the example you gave. There is only 1 policy snapshot object for a snapshot of the CRP placement policy.